### PR TITLE
Create images for Windows 1903

### PR DESCRIPTION
**Pull request by jballe**
This is just the Sitecore 9.1.1 ltsc2019 images using 1903 instead

### CODE REVIEW

File: images/9.1.1 rev. 002459/windowsservercore-1903/sitecore-xm1-cm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    Expand-Archive -Path (Join-Path $env:INSTALL_TEMP '*Configuration files*.zip') -DestinationPath $env:INSTALL_TEMP; `
+    $config = Get-Content (Join-Path $env:INSTALL_TEMP $env:SIF_CONFIG) | ConvertFrom-Json; `
+    $config.Tasks.InstallWDP.Params.Arguments | Add-Member -Name 'Skip' -Value @(@{'ObjectName' = 'dbDacFx'}, @{'ObjectName' = 'dbFullSql'}) -MemberType NoteProperty; `
+    $config.Variables.'Site.PhysicalPath' = 'C:\inetpub\{0}' -f $env:SITENAME; `
+    ConvertTo-Json $config -Depth 50 | Set-Content -Path (Join-Path $env:INSTALL_TEMP $env:SIF_CONFIG); `
+    Rename-Item -Path (Resolve-Path (Join-Path $env:INSTALL_TEMP $env:SIF_CONFIG)).Path -NewName 'config.json'; `
+    Rename-Item -Path (Resolve-Path (Join-Path $env:INSTALL_TEMP $env:SIF_PACKAGE)).Path -NewName 'package.zip';
+
+# Runtime
+FROM sitecore-base:9.1.1-windowsservercore-ltsc2019 as final
```
Comment (pbering) [inline images/9.1.1 rev. 002459/windowsservercore-1903/sitecore-xm1-cm/Dockerfile]: Change `ltsc2019` to `1903`

File: images/9.1.1 rev. 002459/windowsservercore-1903/sitecore-xm1-cm/build.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "tag": "sitecore-xm1-cm:9.1.1-windowsservercore-ltsc2019",
```
Comment (pbering) [inline images/9.1.1 rev. 002459/windowsservercore-1903/sitecore-xm1-cm/build.json]: Change `ltsc2019` to `1903`

File: images/9.1.1 rev. 002459/windowsservercore-1903/sitecore-xm1-cm/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# escape=`
+FROM mcr.microsoft.com/windows/servercore:ltsc2019 as builder
```
Comment (pbering) [inline images/9.1.1 rev. 002459/windowsservercore-1903/sitecore-xm1-cm/Dockerfile]: Change `ltsc2019` to `1903`

**Review by pbering — CHANGES_REQUESTED** [review]
Thanks! Left you a few comments on places that needs to be changed from `ltsc2019` to `1903`

**Comment by jballe** [comment]
Thanks I have fixed those and also added openjdk

**Review by pbering — APPROVED** [review]

File: tests/9.1.1 rev. 002459/1903/docker-compose.xm.sxa.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+version: '2.4'
```
Comment (pbering) [inline tests/9.1.1 rev. 002459/1903/docker-compose.xm.sxa.yml]: File should be removed as the variants is not yet on 1903

File: tests/9.1.1 rev. 002459/1903/docker-compose.xm.pse.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+version: '2.4'
```
Comment (pbering) [inline tests/9.1.1 rev. 002459/1903/docker-compose.xm.pse.yml]: File should be removed as the variants is not yet on 1903

File: tests/9.1.1 rev. 002459/1903/docker-compose.xm.jss.yml [hunk 6]
```diff
@@ -0,0 +1,39 @@
+version: '2.4'
```
Comment (pbering) [inline tests/9.1.1 rev. 002459/1903/docker-compose.xm.jss.yml]: File should be removed as the variants is not yet on 1903

File: tests/9.1.1 rev. 002459/1903/docker-compose.xp.jss.yml [hunk 7]
```diff
@@ -0,0 +1,68 @@
+version: '2.4'
```
Comment (pbering) [inline tests/9.1.1 rev. 002459/1903/docker-compose.xp.jss.yml]: File should be removed as the variants is not yet on 1903

File: tests/9.1.1 rev. 002459/1903/docker-compose.xp.pse.yml [hunk 8]
```diff
@@ -0,0 +1,68 @@
+version: '2.4'
```
Comment (pbering) [inline tests/9.1.1 rev. 002459/1903/docker-compose.xp.pse.yml]: File should be removed as the variants is not yet on 1903

File: tests/9.1.1 rev. 002459/1903/docker-compose.xp.sxa.yml [hunk 9]
```diff
@@ -0,0 +1,68 @@
+version: '2.4'
```
Comment (pbering) [inline tests/9.1.1 rev. 002459/1903/docker-compose.xp.sxa.yml]: File should be removed as the variants is not yet on 1903

**Review by pbering — CHANGES_REQUESTED** [review]
A few comments on compose test files that needs to be removed

**Comment by pbering** [comment]
also it seems that there are two dumps of Sitecore.config in the root named *.xml

**Comment by jballe** [comment]
Oh yes, I actually just copied the folder, but I have not touched the variants so the tests should also not be there. I have removed them now

**Review by pbering — APPROVED** [review]

**Pull request merged**